02_knight_and_princess/tamara_gadieva/task2.hpp

```diff
@@ -0,0 +1,38 @@
+#include <cstdlib>
+#include <iostream>
+
+#define VERTEX_NUM 14
```

Comment:
Как уже обсуждалось, макросы - это зло. Вместо них используем `constexpr` константы
```suggestion
constexpr int VERTEX_NUM = 14;
```